src/CMakeLists.txt

```diff
@@ -83,6 +83,12 @@ if(${TRITON_ENABLE_HTTP} OR ${TRITON_ENABLE_METRICS} OR
   message(STATUS "Using libevent ${Libevent_VERSION}")
 endif()
 
+if(${TRITON_ENABLE_KAFKA})
+  set(LIBRDKAFKA_INCLUDE_DIR /usr/local/include)
```

Comment:
do you need some kind of `find_package(modern-cpp-kafka ...)` here like the HTTP and GRPC endpoints do?

Reply:
I feel like the Kafka change involves multiple repos, can you link all PRs that are needed for Kafka change

Reply:
Added to the PR comment: https://github.com/triton-inference-server/third_party/pull/21